README.md

```diff
@@ -1,4 +1,13 @@
-# greenstar
+# GreenSTAR
+
+[![Build Status](https://github.com/arikkfir/greenstar/actions/workflows/build.yaml/badge.svg)](https://github.com/arikkfir/greenstar/actions/workflows/)
```

Comment:
This needs updating as that workflow has been renamed to `e2e-tests.yaml`.
I also suggest we add `linter.yaml` and `unit-tests.yaml` as well.